### PR TITLE
Fixed boss not moving up after charged shot <bugfix>

### PR DESCRIPTION
**Pull request by Icety**

### CODE REVIEW

File: src/main/java/application/core/aliens/FinalBoss.java [hunk 1]
```diff
@@ -140,17 +141,20 @@ public void setSecondShot(int tSecondShot) {
         this.tSecondShot = tSecondShot;
     }
 
-    protected void changeImageY(boolean charge) {
```
Comment (tieswesterborg) [inline src/main/java/application/core/aliens/FinalBoss.java]: Missing JavaDoc.

File: src/main/java/application/core/aliens/FinalBoss.java [hunk 2]
```diff
@@ -140,17 +141,20 @@ public void setSecondShot(int tSecondShot) {
...
-        if (charge) {
+    protected void changeImageY() {
```
Comment (tieswesterborg) [inline src/main/java/application/core/aliens/FinalBoss.java]: Missing JavaDoc

**Comment by dvantetering** [comment]
Looks good, tip for next time: provide a description and add JavaDoc for better readability. Ready for merging.

**Comment by tieswesterborg** [comment]
Looks Good, just add one JavaDoc please

**Pull request merged**